infra/Pos/Communication/Limits/Types.hs

```diff
@@ -120,6 +121,9 @@ newtype LimitedLengthExt s l a = LimitedLength
     { withLimitedLength :: a
     } deriving (Eq, Ord, Show)
 
+withLimitedLength' :: Proxy s -> LimitedLengthExt s l a -> a
+withLimitedLength' _ = withLimitedLength
```

Comment:
Most of the people are already convinced that `withLimitedLength @s` looks better than `withLimitedLength' (Proxy @s)`.